core/eolearn/tests/test_eodata.py

```diff
@@ -88,17 +91,19 @@ def test_vector_feature_types() -> None:
     "invalid_entry", [0, list(range(4)), tuple(range(5)), {}, set(), [1, 2, 4, 3, 4326, 3], "BBox"]
 )
 def test_bbox_feature_type(invalid_entry: Any) -> None:
-    eop = EOPatch()
     with pytest.raises((TypeError, ValueError)):
-        # Invalid bbox entry should raise an error
+        EOPatch(bbox=invalid_entry)
+
+    eop = EOPatch(bbox=DUMMY_BBOX)
+    with pytest.raises((TypeError, ValueError)):
         eop.bbox = invalid_entry
 
 
 @pytest.mark.parametrize(
     "valid_entry", [["2018-01-01", "15.2.1992"], (datetime.datetime(2017, 1, 1, 10, 4, 7), datetime.date(2017, 1, 11))]
 )
 def test_timestamp_valid_feature_type(valid_entry: Any) -> None:
-    eop = EOPatch()
+    eop = EOPatch(bbox=DUMMY_BBOX, timestamps=valid_entry)
     eop.timestamps = valid_entry
```

Comment:
can be removed 

```suggestion
```

Reply:
why should it be removed? It checks that no error is raised if it's assigned to the attribute (different mechanism kicks AFAIK)

Reply:
I assumed the attribute assignment is tested elsewhere, here you just moved the timestamps assignment into the EOPatch init. I guess it doesn't hurt, but if this is already tested elsewhere, perhaps the flow of the test becomes weird.

Reply:
love your faith in our test suite :)
but no, this is the only place. However now that i think of it, since the `__init__` is autogenerated by attrs, it could be removed from the initialization :thinking: But that is kinda white-box testing.